### PR TITLE
Update projects and other content

### PR DESCRIPTION
**Pull request by Niyousha99**
- Updated homepage statistics and final showcase event recap
- Added board section to teams page and removed outgoing execs
- Updated showcase winners section and added this year's projects to projects page
- Updated exec application form
- Other minor refactors

### CODE REVIEW

**Comment by Niyousha99** [comment]
There is a bug with the past projects carousel not displaying in mobile. Merge blocked until this is resolved

File: src/components/TeamSection/TeamSection.tsx [hunk 1]
```diff
@@ -25,15 +24,15 @@ const TeamSection = () => {
           onClick={() => setToggled(prev => !prev)}
         />
         <S.ToggleButtonLabel htmlFor="checkbox">
-          <S.PMText>
+          <S.TeamCategoryText>
             Project Managers &emsp; &emsp; &emsp;&emsp;&emsp; Executives
             <S.Slider></S.Slider>
```
Comment (Anidion) [inline src/components/TeamSection/TeamSection.tsx:29]: The shadow for the selected element is a bit low for this Slider in mobile view (desktop is perfect)
image

File: src/pages/ApplyPage.tsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { motion } from "framer-motion";
 import Lottie from "react-lottie";
 import "./ApplyPage.css";
 import { applyPageTeamLottieOptions } from "utility/LottieOptions";
-import final_showcase_2023 from "images/final_showcase_teams.jpg";
+import final_showcase_2023 from "images/execs_24.jpg";
```
Comment (Anidion) [inline src/pages/ApplyPage.tsx:7]: Minor but we should rename the variable too
```suggestion
import execs_24 from "images/execs_24.jpg";
```

**Review by Anidion — APPROVED** [review]
LGTM

**Review by AarshShah9 — CHANGES_REQUESTED** [review]
LGTM other than the shadow on the button that Ben mentioned

**Comment by brian-ngyn** [comment]
LGTM

**Review by brian-ngyn — APPROVED** [review]

**Comment by Niyousha99** [comment]
> There is a bug in this PR with the past projects carousel not displaying in mobile. Merge blocked until this is resolved

Fixed

File: src/components/TeamSection/TeamSection.tsx (hunk 1 above)
Comment (Niyousha99) [reply src/components/TeamSection/TeamSection.tsx:29]: Good catch! This should be fixed now, please double check

File: src/pages/ApplyPage.tsx (hunk 2 above)
Comment (Niyousha99) [reply src/pages/ApplyPage.tsx]: Done!

**Pull request merged**